src/test/java/io/jenkins/plugins/services/WikiServiceTest.java

```diff
@@ -53,7 +56,10 @@ public void testGetWikiContentGitCustom() {
 
   public void testGetWikiContentGit(String url) {
     System.setProperty("github.client.id", "dummy");
-    final String content = wikiService.getWikiContent(url);
+    final String content = wikiService.getWikiContent(new Plugin() {
+      @Override public String getWikiUrl() { return url; }
+      @Override public String getDefaultBranch() { return "master"; }
```

Comment:
Perhaps it's a better look to use 
```suggestion
      @Override public String getDefaultBranch() { return "main"; }
```
here (that's the new GH default anyway) + below, and to use a sample plugin that uses a `main` default branch?